spec/support/Cohortmanager.spec.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars */
+import Cohortmanager from '../../src/Cohortmanager.js'
+describe('Cohortmanager', () => {
+  // eslint-disable-next-line no-unused-vars
+  let cohortmanager
+  beforeEach(() => {
+    cohortmanager = new Cohortmanager()
+  })
+  describe('creates, search and removes a cohort by cohort name', () => {
```

Comment:
I would personally split each of these verbs into different describe blocks as you're testing different things with each of them. You can then have different cases for each of them, eg cohort found, cohort not found, bad input etc.